docs/02-get-started/08-enable-disable-upgrade-kyma-module.md

```diff
@@ -2,7 +2,9 @@
 title: Enable, disable and upgrade a Kyma module
 ---
 
-Learn how to enable, disable and upgrade a Kyma module. To see the list of all available and planned Kyma modules, go to [Overview](../../01-overview/README.md).
+This guide shows how to quickly install, uninstall or remove Kyma with specific modules. To see the list of all available and planned Kyma modules, go to the [Kyma modules](../README.md#kyma-modules).
+
+> **NOTE:** This guide describes installation of a standalone Kyma with specific modules, and not how to enable modules in SAP BTP, Kyma runtime (SKR).
```

Comment:
```suggestion
> **NOTE:** This guide describes installation of a standalone Kyma with specific modules. If you are using SAP BTP, Kyma runtime (SKR), read [Enable and Disable a Kyma Module](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module?locale=en-US&version=Cloud) instead.
```

Instead of saying what this guide does not do, we could point the reader to the document that does it.

Reply:
good point Nina, I was thinking about that, @pbochynski is there anything blocking or preventing us from doing that?